themes/theme-ios11/pages/Scanner/index.jsx

```diff
@@ -42,17 +42,15 @@ class ScannerView extends PureComponent {
       <View background={backgroundColor}>
         <RouteContext.Consumer>
           {({ query: { scope = SCANNER_SCOPE_DEFAULT, type = SCANNER_TYPE_BARCODE } = {} }) => (
-            <Fragment>
+            <ScannerContainer
+              scope={scope}
+              type={type}
+              scannerDidOpen={this.removeBackground}
+              scannerDidClose={this.resetBackground}
+            >
               <BackBar title="titles.scanner" right={null} />
```

Comment:
This change will have the effect that if the scanner support is deactivated by config (and/or by lib version) but the user gets redirected to this page, he will be stuck there because the children in this container are only rendered when the scanner support is enabled.